### PR TITLE
Addition of "Open File" button

### PR DESCRIPTION
**Pull request by heartsinaustin**
Allows user to open a local python file.

### CODE REVIEW

File: PyPlay.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ document.addEventListener("DOMContentLoaded", () => {
     ace.require("ace/ext/language_tools");
     const editor = ace.edit("editor");
     editor.session.setMode("ace/mode/python");
+	editor.setShowPrintMargin(false);
```
Comment (ryanpcmcquen) [inline PyPlay.js]: Looks like the indentation is off, can you set up your editor to use Prettier?

Comment (heartsinaustin) [reply PyPlay.js]: Ok I switched editors to atom and installed Prettier.  It seems to modify the file correctly.  Going to see if I can push the changes now.

Comment (heartsinaustin) [reply PyPlay.js]: Sorry I am not sure if I "pushed" the changes correctly.  Please let me know if I did not.

Comment (ryanpcmcquen) [reply PyPlay.js]: I don't see any additional commits.

**Comment by heartsinaustin** [comment]
Sorry I realized that while I can make the open file dialog work It breaks the encoder.

**Comment by ryanpcmcquen** [comment]
You can keep it open and keep working on it.

**Pull request closed without merging**